pyccl/core.py

```diff
@@ -197,6 +198,7 @@
     'eisenstein_hu':    lib.eisenstein_hu,
     'bbks':             lib.bbks,
     'boltzmann_class':  lib.boltzmann_class,
+    'boltzmann_camb':   lib.boltzmann_camb,
```

Comment:
@c-d-leonard The thing you saw is def not a problem on master. I think you didn't fully reinstall and remove old files.

Reply:
Yep you're right, my mistake.